### ISSUE TITLE
[Google Ads Transparency Center API] Get direct link to the video for video format results

### ISSUE THREAD

**Issue by martin-serpapi** [serpapi/public-roadmap#1231]
A customer requested to retrieve the actual video instead of a raw JavaScript for results that have a video format:

![image](https://github.com/serpapi/public-roadmap/assets/144151895/9e95fd6f-84e9-472e-9807-7ad1793d702f)

[Playground](https://serpapi.com/playground?engine=google_ads_transparency_center&text=apple.com&region=2840) | [Inspector](https://serpapi.com/searches/6564ce8d5119a65173c7bda3/inspect)

The `link` URL leads to a JavaScript code response:

https://displayads-formats.googleusercontent.com/ads/preview/content.js?client=ads-integrity-transparency&creativeId=680931481197&obfuscatedCustomerId=9183552685&uiFeatures=12,54&adGroupId=158700116867&itemIds=6167587766021709499,6763212636745958213,17677254667565431111,5582486791605524854,5782058483052026799&allowedVariations=44&overlay=%3DH4sIAAAAAAAAAPNS4OJPrShJLcpLzAnLTEnN90wR4uXiLjWt8PczzU63yAsFAFsyntQiAAAA&sig=ACiVB_zYSnC8cmwpLAHjIn-6BtDzXcbYTA&htmlParentId=fletch-render-16509338303855644191&responseCallback=fletchCallback16509338303855644191

[Intercom](https://app.intercom.com/a/inbox/yzck75jr/inbox/shared/all/conversation/102910000054315?view=List)

**Comment by hartator** [comment]
@djsegal seems like a lot of work indeed.

Let's froze this.

**Comment by tanys123** [comment]
@djsegal Indeed, I have a working code in the initial engine [PR](https://github.com/serpapi/SerpApi/pull/4176), probably that require a new engine.

**Comment by schaferyan** [comment]
Another customer requested this:

[Intercom](https://app.intercom.com/a/inbox/yzck75jr/inbox/shared/all/conversation/102910000055119?view=List)

**Comment by tanys123** [comment]
We provided a workaround in a [blog post](https://serpapi.com/blog/retrieving-a-direct-media-link-from-the-google-ads-transparency-center-api). Feel free to check it out to see if it works with your workflows.
